math/radians.msl

```diff
@@ -0,0 +1,15 @@
+#include "const.msl"
+
+/*
+contributors: Patricio Gonzalez Vivo, Anton Marini
+description: radians polyfill
+use: <float> radians(<float>) 
+license:
+    - Copyright (c) 2021 Patricio Gonzalez Vivo under Prosperity License - https://prosperitylicense.com/versions/3.0.0
+    - Copyright (c) 2021 Patricio Gonzalez Vivo under Patron License - https://lygia.xyz/license
+*/
+
+#ifndef FNC_RADIANS
+#define FNC_RADIANS
+inline float radians(float angle) { return angle * PI / 180.0; }
```

Comment:
We already have a DEG2RAD constant in const.glsl. Shouldn't we keep this consistent in Metal?